code/modules/mob/new_player/preferences_setup.dm

```diff
@@ -118,32 +118,88 @@
 
 
 /datum/preferences/proc/update_preview_icon()
-	if(!SSatoms.initialized)
-		return // clothing wont work properly
-		
-	// Set up the dummy for its photoshoot
-	var/mob/living/carbon/human/dummy/mannequin = generate_or_wait_for_human_dummy(DUMMY_HUMAN_SLOT_PREFERENCES)
-	copy_to(mannequin)
-
 	// Determine what job is marked as 'High' priority, and dress them up as such.
 	var/datum/job/previewJob
+	var/highest_pref = JOBS_PRIORITY_NEVER
```

Comment:
This is 4, not 0, and will break the logic.

Reply:
I'll just swap the defines since /tg/ uses it reversed from us